main.go

```diff
@@ -92,6 +92,11 @@ func main() {
 			EnvVar: "PLUGIN_HUGO_VERSION",
 			Value:  "",
 		},
+		cli.BoolFlag{
+			Name:   "hugoExtended",
+			Usage:  "If the hugo extended package should be used",
+			EnvVar: "PLUGIN_HUGO_EXTENDED",
```

Comment:
Maybe just `PLUGIN_EXTENDED`? It's already obvious enough that you are using hugo with something like that:

```
- name: deploy
  image: plugins/hugo
  settings:
    extended: true
```